backend/apps/tickets/views.py

```
@@ -57,6 +57,13 @@ def get_categories(request):
 def get_priorities(request):
     return TicketPriority.objects.all()
 
+@router.get("/community", response=list[TicketSchema])
+def community_tickets(request):
```

Ruff (0.15.0)

[warning] Unused function argument: `request`

(ARG001)